lib/plugins/config/admin.php

```diff
@@ -77,24 +77,26 @@ public function html()
         $allow_debug = $GLOBALS['conf']['allowdebug']; // avoid global $conf; here.
         global $lang;
         global $ID;
+		
+		define('kHeadlineLevel', '2'); /* change this if doc structure requires */
```

Comment:
Why not use a `const` at the class level instead of a `define(...)` call

Reply:
I would prefer `const` if it could be used within a `class` member. Maybe in PHP 9... :-/